proposals/0000-proposal-process.rst

```diff
@@ -0,0 +1,319 @@
+   `provided template <https://github.com/ghc-proposals/ghc-proposals/blob/master/0000-template.rst>`_)
+   specifying the proposed change. This document is the primary specification
+   of the feature and takes the place of the current Trac Wiki document.
+   
+2. Community members (including members of the proposal committee) will discuss
+   the proposal. The submitter is responsible for amending the specification to
+   account for collected comments. It should be expected that the proposal will
+   change during this process, which may last from days to months.
+
+3. When the author feels that the proposal is complete, and not being improved
+   by further discussion, they can submit the proposal to the proposal committee
+   for a decision.
+
+4. Within two (exceptionally three) weeks the committee will arrive at a
+   consensus as to whether the proposal in its submitted form meets a set of
+   criteria for inclusion (see below) while weighing the opinions expressed by
+   the community.
+
+   If the proposal is rejected then it will be sent back to the submitter along
+   with a rationale (referencing the criteria defined below). This is by no
+   means a final decision; the submitter may at any time amend and re-submit the
+   proposal to address the concerns of the committee.
+
+4. When the proposal is accepted the pull request will be merged and the
+   document will be preserved in the proposals repository as a permanent
+   specification for the feature.
+   
+5. The author will create a Trac ticket linking to the proposal to
+   provide a place track the progress of the implementation.
+
+6. The author may choose to implement the proposal after acceptance, but she is
+   under no obligation to do so. (Of course, neither is anyone else.)
+
+7. During implementation, it is very likely that the specification will be refined.
+   The implementor should keep the specification up to date over the course of
+   the implementation.
+
+Since the proposal pages already existing on the Trac `Wiki
+<https://ghc.haskell.org/trac/ghc/wiki/Proposal>`_ represent a significant
+amount of effort and knowledge, we'll make an effort to import those which have
+been implemented into the proposals repository if this scheme is adopted.
+
+GHC Committee
+~~~~~~~~~~~~~
+
+The process involves forming a small group which is responsible for
+deciding whether proposed changes should be accepted after discussion
+within the community. 
+
+The committee should be large enough to reflect the diversity of GHC's
+contributor- and user-base but small enough to ensure a sense of individual
+responsibility among its members. A size of six to eight members would likely be
+a good starting point.
+
+To simplify the committee selection process, I propose that the committee is
+chosen as follows. We seek nominations (including self-nominations); and then
+Simon Peyton-Jones and Simon Marlow select members from those nominations. This
```

Comment:
*Meta:* I think we can also comment using line-comments. And we can *react* on those.

E.g. I also agree that Simon&Simon is a good combo to act as a chair.